recipes/r-gsw/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "1.0-3" %}
+
+package:
+  name: r-gsw
+  version: {{ version.replace("-", "_") }}
+
+source:
+  fn: gsw_{{ version }}.tar.gz
+  url:
+    - http://cran.r-project.org/src/contrib/gsw_{{ version }}.tar.gz
+    - http://cran.r-project.org/src/contrib/Archive/gsw/gsw_{{ version }}.tar.gz
+  sha256: e24c6802467546d616531499c2a98610fdb5c35c8a1e161b6d6b887c4c814331
+
+build:
+  number: 0
+  skip: True  # [win32]
+  script: R CMD INSTALL --build .
```

Comment:
Should be fine. I use `conda skeleton cran` to generate all of the R recipes.

A not entirely insignificant amount of them need some customization of the build process, and that's easier done in a separate file.

I would not be opposed to changes to `conda skeleton cran` so that it reads existing recipes and looks for customized build scripts and if found uses them otherwise removes them and replaces them with `script:` instead.